packages/components/stamp/src/stamp.tsx

```diff
@@ -40,63 +41,72 @@ const getPaddingStyle = (props: Props) => {
   `;
 };
 
-const getToneStyles = (props: Props) => {
+const getToneStyles = (props: Props & { isNewTheme: boolean }) => {
   switch (props.tone) {
     case 'critical': {
       return css`
         background-color: ${designTokens.colorError95};
-        border: 1px solid ${designTokens.colorError};
+        border: 1px solid
+          ${props.isNewTheme && designTokens.borderForStampWhenError};
```

Comment:
I think we don't need the `props.isNewTheme` here:
1. the design token will make the ui to look different based on the selected theme
2. when its value is `false`, the value of the color for the border would be the string `false`